core/impl/src/main/java/pt/webdetails/cdf/dd/model/meta/reader/datasources/DataSourcesModelReader.java

```diff
@@ -74,87 +81,99 @@ private void readDataSourceComponent( MetaModel.Builder model, Pointer pointer,
     Map<String, Object> def = (Map<String, Object>) pointer.getNode();
     JXPathContext jctx = JXPathContext.newContext( def );
 
+    String name = pointer.asPath().replaceAll( ".*name='(.*?)'.*", "$1" );
     String label = (String) jctx.getValue( "metadata/name" );
 
-    String dataSourceType = (String) jctx.getValue( "metadata/datype" );
+    String category = (String) jctx.getValue( "metadata/group" );
+    String categoryLabel = (String) jctx.getValue( "metadata/groupdesc" );
 
-    boolean isCPK = dataSourceType.equalsIgnoreCase( "cpk" );
-    boolean isCDA = !isCPK;
+    String dataAccessType = (String) jctx.getValue( "metadata/datype" );
```

Comment:
Actually, although `datype` comes from Data Access Type — a CDA concept —, it was generalized, with time, becoming the type of data source. Hence, its use for data sources other than "CDA", like "CPK". So I argue that this is a good rename of the original variable name, `dataSourceType`.